app/Http/Resources/Manager/CourseIndexResource.php

```diff
@@ -22,11 +23,43 @@ class CourseIndexResource extends JsonResource
     #[\Override]
     public function toArray(Request $request): array
     {
+        $course = $this->resource;
+
         return [
             ...(new CourseResource($this->resource))->toArray($request),
             'instructor' => new InstructorResource($this->resource->instructor),
             'has_active_students' => $this->resource->attendances()->exists(),
             'tags' => TagResource::collection($this->resource['tags']),
+            'deadline' => $this->buildDeadlinePayload($course),
         ];
     }
+
+    /**
+     * 期限情報（none / fixed_date / relative_days）を統一フォーマットで生成
+     *
+     * @return array{deadline_type:string,fixed_date:?string,relative_days:?int}
+     */
+    private function buildDeadlinePayload(Course $course): array
+    {
+        $enum = DeadlineTypeEnum::tryFrom($course->deadline_type) ?? DeadlineTypeEnum::NONE;
+        $rel  = $course->courseDeadline;
+
+        return match ($enum) {
+            DeadlineTypeEnum::FIXED_DATE => [
+                'deadline_type' => $enum->value,
+                'fixed_date'    => $rel?->fixed_date ? (string) $rel->fixed_date : null,
+                'relative_days' => null,
+            ],
+            DeadlineTypeEnum::RELATIVE_DAYS => [
+                'deadline_type' => $enum->value,
+                'fixed_date'    => null,
+                'relative_days' => is_null($rel?->relative_days) ? null : (int) $rel->relative_days,
+            ],
+            default => [
+                'deadline_type' => DeadlineTypeEnum::NONE->value,
+                'fixed_date'    => null,
+                'relative_days' => null,
+            ],
+        };
```

Comment:
`deadline_type`は講座に関するフィールドなので、排除するようにお願いします！

また、不整合データが生成されない限りは、値を参照するだけで問題ないので、次のようなイメージでコードの簡素化をお願いしたいです！

```suggestion
        return [
            'course_deadline_id' => $this->resource->id,
            'fixed_date' => $this->resource->fixed_date?->format('Y-m-d'),
            'relative_days' => $this->resource->relative_days,
       ];
```